### PR TITLE
Ubuntu 16.04

### PR DESCRIPTION
**Pull request by ramereth**
I've reverted the previous builds that were having issues and also made a few enhancements.

### CODE REVIEW

**Comment by ramereth** [comment]
[Ubuntu 16.04 BE & LE template fix up to pass openstack_taster](https://trello.com/c/DGb18Fdt/487-ubuntu-1604-be-le-template-fix-up-to-pass-openstacktaster)

**Comment by ramereth** [comment]
@osuosl/systems please take a look when you get a chance

**Comment by doublej472** [comment]
BE has a failure trying to load a specific module, I don't know if it actually affects anything though:
```
Dec 20 20:46:03 unassigned-hostname systemd-modules-load[380]: Failed to find module 'ibmvscsic'
```

File: scripts/ubuntu/sudoers.sh [hunk 1]
```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
+sed -i -e '/Defaults\s\+env_reset/a Defaults\texempt_group=sudo' /etc/sudoers;
+
+# Set up password-less sudo for the ubuntu user
+echo 'ubuntu ALL=(ALL) NOPASSWD:ALL' >/etc/sudoers.d/99_ubuntu;
+chmod 440 /etc/sudoers.d/99_ubuntu;
```
[the remark this answers is not in the document]
Comment (ramereth) [reply scripts/ubuntu/sudoers.sh]: I am pretty sure that cloud-init supplies this on boot and isn't generated in the image initially.

File: scripts/ubuntu/cleanup.sh [hunk 2]
```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+
+# Delete X11 libraries
+apt-get -y purge libx11-data xauth libxmuu1 libxcb1 libx11-6 libxext6;
+
+# Delete oddities
+apt-get -y purge popularity-contest installation-report friendly-recovery laptop-detect;
```
[the remark this answers is not in the document]
Comment (ramereth) [reply scripts/ubuntu/cleanup.sh]: I think these are safe to keep. They aren't taking up much space FWIW.

**Comment by ramereth** [comment]
@doublej472 I'd prefer to keep the default fstab options if possible. If people really want that boost they can adjust that themselves.

File: scripts/ubuntu/sudoers.sh (hunk 1 above)
[the remark this answers is not in the document]
Comment (ramereth) [reply scripts/ubuntu/sudoers.sh]: In case there's an issue on boot and cloud-init can't talk to the metadata server.

**Pull request merged**